kernel/src/arch/x86_64/syscall.rs

```diff
@@ -258,10 +259,15 @@ fn handle_vspace(arg1: u64, arg2: u64, arg3: u64) -> Result<(u64, u64), KError>
     let mut p = kcb.arch.current_executor()?;
 
     match op {
-        VSpaceOperation::Map => unsafe {
+        VSpaceOperation::Map | VSpaceOperation::MapPM => unsafe {
```

Comment:
maybe we should rename those enum variants to MapMem and MapPMem now?

Reply:
Done.